### ISSUE TITLE
usecase assigned & derived categorisation

### ISSUE THREAD

**Issue by bertvannuffelen** [dpvcg/personal-data#2]
Hi,

How do one handle the case, that a responsible wants to know in the same knowledge base whether the category has been assigned by hand, or has been derived by a machine.

This is important as the first case is the consequence of a human action (e.g. court case, request by user, ...) as a kind of overruling. Whereas the second case is what defines the behavior of the system that would automatically act on these categorization. 

So how do I make a distinction between the two cases using the hereunder defined ontology?

kr,

Bert

**Comment by coolharsh55** [comment]
I'm not sure I understand the question correctly, so I'm answering based on what I understand - you need to distinguish between categories that are the result of a human-centric action (i.e. created or produced from an action by a human) vs machine-generated (i.e. created or generated by a machine). 

Is it not possible to simply create two classifications - called generated_by_human and generated_by_machine and then apply these where relevant in your specific use-case? I do not think that any category can be stated to always be generated by machines or humans.

**Comment by bertvannuffelen** [comment]
On the usecase:

_example story_
Bob buys a new mobile phone. He registers it for warrenty. And to get an extra bonus some addition info. Based on that additional info, he is clustered as a banana-lover.  Therefore he gets all the time info around banana's. Bob takes action and informs the company that he does not want anymore info around banana's, but that monkeys are ok. The operator enters this info in the system. 

In the example, the clustering algorithm is machine derived personal data, while the info entered by the operation is human derived data.   

On your suggestion:
I follow your approach. But in many case there might be dependencies between the combinations (e.g. clustering is only  a machine derived data, while court_enforcement is a human derived data.)  
If I follow your approach,  do I have to create a new ontology where I have to express all combinations as subclasses from both dimensions?

kr,

Bert

**Comment by coolharsh55** [comment]
Yes, you have to create or extend the ontology for your specific use-case to fit your requirement. If you want *every* data category to be classified as either machine-derived or human-generated, then you would create two top-level classes representing each and use them as parent classes.
